spec/acceptance/iis_application_spec.rb

```diff
@@ -108,7 +108,7 @@
                 'bindinginformation'   => '*:443:#{site_hostname}',
                 'protocol'             => 'https',
                 'certificatestorename' => 'MY',
-                'certificatehash'      => '#{thumbprint}',
+                'certificatehash'      => '#{thumbprint.downcase}',
```

Comment:
IIS accepts both upper and lower cases, but only returns one of them. IIRC it returns lowecase in the gui and uppercase in powershell.

Maybe a better fix would be to compare the passed thumbprint to both upper and lowercase versions of whatever IIS has, but just upcasing it was the easiest.